src/LiveQuery/SessionTokenCache.js

```diff
@@ -27,14 +29,22 @@ class SessionTokenCache {
     if (!sessionToken) {
       return Parse.Promise.error('Empty sessionToken');
     }
-    const userId = this.cache.get(sessionToken);
-    if (userId) {
-      logger.verbose('Fetch userId %s of sessionToken %s from Cache', userId, sessionToken);
-      return Parse.Promise.as(userId);
+
+    if (this.cache.has(sessionToken)) {
```

Comment:
This logic may be problematic, if we need to invalidate a session we should mark it INVALID_SESSION_TOKEN.

Also I’m not sure that the solution is the right one as every time a session will be missing, we’ll interpret as invalid. I’d rather add a Boolean onlyFromCache = false of we don’t want to poke the server.